### PR TITLE
Added a test for suppressed mouse events on canceled pointerdown

### LINKED ISSUE #3189: Need to test that canceling a pointerdown prevents compat mouse events
A canceled pointerdown suppresses compat mouse events until the next pointerup. We need to test this.

### PR DESCRIPTION
**Pull request by mustaqahmed**
Closes #3189

### CODE REVIEW

**Comment by mustaqahmed** [comment]
ptal, resolved the sync problem in my branch.

**Comment by RByers** [comment]
Looks reasonable to me.  Is it worth also adding a "drag" case so we can verify that no `mousemove` is generated?  Obviously it would have to be robust to the number of move events - so maybe append to the log at most once for move events?

**Comment by mustaqahmed** [comment]
I tried exactly that in my first commit. The problem is that neither Edge nor Chrome fires any mouse events on touch drag, so the no-canceled pointerdown case fails.

Does it make sense to split the test into touch and non-touch case?

File: pointerevents/pointerevent_compatibility_events_on_prevent_default-manual.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+          showPointerTypes();
+      }
+
+      function end_of_interaction() {
+          test(function () {
+              assert_equals(event_log.join(", "),
+                        "pointerdown@target0, " +
+                        "pointerup@target0, " +
+                        "pointerdown@target1, mousedown@target1, " +
+                        "pointerup@target1, mouseup@target1");
```
Comment (scottgonzalez) [inline pointerevents/pointerevent_compatibility_events_on_prevent_default-manual.html]: These lines are indented one too many times.

Comment (mustaqahmed) [inline pointerevents/pointerevent_compatibility_events_on_prevent_default-manual.html]: Done, ptal.

Also split the test into two cases: click/tap (for any pointer type) and drag (only for mouse).

File: resources [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit dee0f92b957973785097af3e1b1a5ded9b1e7a94
+Subproject commit f8de1efce69fe125846c2ae621e557947899413e
```
Comment (RByers) [inline resources]: Looks like this commit includes updates to the repos submodules, that's not intentional is it?

**Comment by RByers** [comment]
Splitting into two tests makes sense to me.  We should be designing our tests for an environment (like Edge, blink and jQuery's harnesses) where they're all automated anyway (since manual tests are of very little value in practice).

There's a lot of boilerplate / duplicated code between theses tests though, but I guess that's a maintenance burden issue for many of the pointer events tests (not necessarily something to be solved in this commit).

File: resources (hunk 2 above)
Comment (mustaqahmed) [inline resources]: Yikes! Fixed.

**Comment by mustaqahmed** [comment]
FYI: both tests are passing on Chrome & Edge.

**Comment by RByers** [comment]
LGTM, thanks

**Pull request merged**